### DISCUSSION TITLE
useNuxtApp, ref... in <script lang="ts">

### DISCUSSION THREAD

**Discussion by nhan-ho-saltovn** [nuxt/bridge#441] [Q&A]
I have a problem when use composition api feature in Typescript file (cannot find name useNuxtApp, ref...) but in Javascript file, everything works perfectly.
Maybe it require some config in ts.config? I am new to Vue and am trying to migrate the old app to Nuxt Bridge. Please help T.T 

![image](https://user-images.githubusercontent.com/109122792/180949055-222c6330-dd80-4e24-b321-e27139c1447a.png)

**Comment by antlionguard** [comment]
Did you set `autoImports: true` in nuxt.config? 

```
...
bridge: {
    ...
    autoImports: true,
    ...
},
...
```

and you need to add this line top of the `tsconfig.json`

```  "extends": "./.nuxt/tsconfig.json",```
